org.gnome.Evince.json

```diff
@@ -13,9 +13,14 @@
         "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
         "--metadata=X-DConf=migrate-path=/org/gnome/evince/",
         "--env=GDK_PIXBUF_MODULE_FILE=/app/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache",
+        "--filesystem=xdg-run/dconf",
```

Comment:
Those should instead be removed downstream...

Reply:
OK. But why? I'm packaging some flatpaks. It will help if I know the reason :)

Reply:
I see https://github.com/flathub/io.gitlab.Goodvibes/issues/8